### ISSUE TITLE
Show GitHub Deployments in Pull Requests

### ISSUE THREAD

**Issue by pcorpet** [Reviewable/Reviewable#590]
I've just discovered an [integration](https://devcenter.heroku.com/articles/github-integration-review-apps) with Heroku that allows to launch my code in a new Herokup app for each Pull Request. This is very useful to get my reviewer to see my code in action (super useful for UI stuff). Below the deck it's using https://developer.github.com/v3/guides/delivering-deployments/ to integrate it, however Reviable does not seem to show the link.

Can you try integrate the button somehow? Maybe with the statuses?

![deployment-screenshot](https://user-images.githubusercontent.com/7937848/45958111-e050b800-c016-11e8-9ad2-dc1d0dcf90ae.png)

**Comment by pkaminski** [comment]
Agreed that this looks useful and is probably best integrated as a variant of statuses.  I'll just need to be careful since the success/failure state of deployments shouldn't have the same effect as that of actual statuses.

I'll definitely add this integration at some point but no promises on scheduling, sorry.

**Comment by pkaminski** [comment]
This could be superseded by #887.
